crates/lib3h/src/engine/ghost_engine.rs

```diff
@@ -216,12 +194,10 @@ impl<'engine> GhostEngine<'engine> {
         let nodes: Vec<Url> = self.config.bootstrap_nodes.drain(..).collect();
         for bs in nodes {
             // can't use handle_bootstrap() because it assumes a message to respond to
-            let cmd = GatewayRequestToChild::Transport(
-                transport::protocol::RequestToChild::SendMessage {
-                    uri: bs,
-                    payload: Opaque::new(),
-                },
-            );
+            let cmd = GatewayRequestToChild::Bootstrap(BootstrapData {
+                space_address: "bogus_space".into(),
```

Comment:
Why is the space bogus?

Reply:
added comment.  this should come from the config, but that space address (equiv to DNA address) isn't there yet.